battery.h

```diff
@@ -12,13 +13,17 @@ class Battery
     void increase_battery(int);
     int get_battery();
     int get_range();
+    void CheckRange();
```

Comment:
Use snake case so it is consistent with everything else:
```suggestion
    void check_range();
```
Make sure to update the `.cpp` file to reflect this change